### PR TITLE
Update Get-ADComputer.md

### PR DESCRIPTION
**Pull request by msbemba**
corrected formatting and added example 6 and 7

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1617

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/addsadministration/Get-ADComputer.md [hunk 1]
```diff
@@ -190,6 +190,28 @@ PS C:\> Get-ADComputer -Filter *
...
+```
+PS C:\> $Date = [DateTime]::Today.AddDays(-30)
+PS C:\> Get-ADComputer -Filter 'Name -like "User01*" -and PasswordLastSet -ge $Date' -Properties IPv4Address | FT Name,DNSHostName,IPv4Address -A
+name        dnshostname            ipv4address
+----        -----------            -----------
+User01-SRV1 User01-SRV1.User01.com 10.194.99.181
+User02-SRV2 User02-SRV2.User02.com 10.194.100.3
+```
+
+This command shows the name, dns hostname, and IPv4 address.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/Get-ADComputer.md]:
```suggestion
This command shows the name, DNS hostname, and IPv4 address.
```
Please also change dns at the end of Example 2.

**Comment by tseward** [comment]
One other edit, can you please change "User01" and "User02" in the cmdlet example to Computer01 and Computer02? This is to prevent confusion on what we're querying. Approved after that.

**Comment by msbemba** [comment]
> One other edit, can you please change "User01" and "User02" in the cmdlet example to Computer01 and Computer02? This is to prevent confusion on what we're querying. Approved after that.

Thank you @tseward  I changed the value for "User01" and "User02" in the cmdlet example to Computer01 and Computer02.

**Comment by e0i** [comment]
> I think you should also change user01 for computer01 on example 7. Also the output should be computer01 and not computer02, I think.

cc @msbemba

**Comment by msbemba** [comment]
> > I think you should also change user01 for computer01 on example 7. Also the output should be computer01 and not computer02, I think.
> 
> cc @msbemba

I have made these changes now.

Thanks
Sri

**Pull request merged**